Main.java

```diff
@@ -1,21 +1,90 @@
 class Main {
   public static void main(String[] args) {
-   // kindly use appropriate data types for your declaration
-   // declare variables days the debtor defaulted, amount the debtor is to pay per day, amount the debtor is to pay, amount the debtor paid and assign values to them
-   
-   //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
-
-   // calculate and print total amount the debtor is to pay
-
-   // calculate and print the days the debtor paid for
-  
-  // calculate and print the amount whose day was not captured because the amount was incomplete
-
-   // calculate and print amount the debtor is left to pay
-
-   // calculate and print days the debtor has not payed for
-
-   //kindly remove the statement below when you are done with the assignment
-    System.out.println("Hello world!");
+    float 	amountToPayDaily = 100.00f;					  //amount debtor opted to pay back per day
+		 float 	DebtOwed = 500.00f;							      //total amount owed by debtor
+		 int 	daysDebtorPaid = 0;							        //number of days debtor made	
+		 int 	daysNotCaptured = 0;						        //number of days that payment was up to $100
+		 float 	amountLeftTopay,amountDebtorPaid;			//amount left to pay //amount debtor has paid
+		 byte 	payBackDays = 5;							        //amount of days debtor chose to pay back	
+		 int 	daysLeftToPay;								          //days left to pay up
+		 float	day1 = 100.00f;  							        //full amount paid on day1
+		 float 	day2 = 100.00f;   							      //full amount paid on day2
+		 float 	day3 = 50.00f;								        //full payment was'nt made
+		 float 	day4 = 60.00f;								        //full payment was'nt made
+		 float 	day5 = 0.00f;								          //no payment has been made
+		 
+		
+		 
+		 			
+		 		System.out.println("Debtor opted-in on the 5 days loan pay back plan");
+		 		
+		 		
+		 
+		 	/*********************************************************************
+		 	 * calculating the  total amount paid by the debtor within the 5 days 
```

Comment:
You are meant to calculate the amount the debtor is to pay here not amount the debtor paid

Reply:
the amount debtor was to pay had already been declared above, with the variable name debtOwed = $500.00